src/components/form/input/input.tsx

```diff
@@ -75,20 +74,14 @@ export const Input: React.FC<InputProps> = ({
   // For checkboxes, this is problematic as it loses the "on" | "off" values.
   // This conditionalizes the presence of the "value" props.
   const valueProps =
-    type.toLowerCase() === "checkbox"
-      ? typeof value === "undefined"
-        ? {}
-        : {
-            value: value,
-          }
-      : {
-          value: valueState,
-        };
+    type.toLowerCase() === "checkbox" || type.toLowerCase() === "radio"
+      ? { checked: props.checked, value: valueState }
```

Comment:
This change seems to break the "on" behaviour as stated above. Is there a reason for this change (and not simply including radio to the check on line 77)?